Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -723,7 +723,7 @@ constexpr LengthType RenderStyle::zeroLength() { return LengthType::Fixed; }
 inline float RenderStyle::zoom() const { return m_nonInheritedData->rareData->zoom; }
 
 // ignore non-standard ::-webkit-scrollbar when standard properties are in use
-inline bool RenderStyle::hasCustomScrollbarStyle() const { return hasPseudoStyle(PseudoId::Scrollbar) && scrollbarWidth() == ScrollbarWidth::Auto; }
+inline bool RenderStyle::hasCustomScrollbarStyle() const { return hasPseudoStyle(PseudoId::Scrollbar) && scrollbarWidth() == ScrollbarWidth::Auto && !scrollbarColor().has_value(); }
```

Comment:
Ah yeah I see what you mean. I guess it's technically correct on the basis that the Legacy styling creates a Custom Scrollbar as opposed to a Native Scrollbar but I agree it can be better. I'll go with hasLegacyScrollbarStyle

Reply:
Actually, even the verb `has` is confusing here, we should use the verb `uses` too.